.github/workflows/ci.yml

```diff
@@ -373,31 +371,23 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: macos-11
-            cxx_compiler: icpc
+          - os: macos-13
+            cxx_compiler: clang++
             std: 17
             build_type: release
             backend: omp
     steps:
       - uses: actions/checkout@v3
-      - name: Install Intel® oneAPI C++ Compiler Classic and Intel® oneAPI Threading Building Blocks
+      - name: Install OpenMP for Clang++
+        if: (matrix.backend == 'omp' && matrix.cxx_compiler == 'clang++')
```

Comment:
Is the if statement here necessary given that it exactly matches the matrix defined on the lines above?

Reply:
Done